scripts/leaderboard.coffee

```diff
@@ -118,6 +156,48 @@ module.exports = (robot) ->
     field[name] = score
     return "#{name}'s score set to #{score}."
 
+  getRank = (score) ->
+    if score <= 5
```

Comment:
It would be better to compare with the pointThresholds array here and below. In that way if the rank ranges need to be changed, only the array needs to ne updated.

Reply:
Okay this will make the rest of the code level agnostic. Adding this.